apps/entry/widgets/date_widget.py

```diff
@@ -22,13 +23,19 @@ def update_attribute(widget, data, widget_data):
 
         'export_data': {
             'data': {
-                'excel': {
+                'version': 1,
+                'common': {
                     'value': date,
+                    'widget_id': WIDGET_ID,
+                    'widget_key': widget.key
+                },
```

Comment:
@thenav56 I have defined meta data here. because during attribute export, it solely relies on `export_data.data`. Not wanting to produce some unwanted side-effects